src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/OperationBindingContext.cs

```diff
@@ -15,9 +15,14 @@ namespace Microsoft.AspNet.Mvc.ModelBinding
     public class OperationBindingContext
     {
         /// <summary>
-        /// Gets or sets the <see cref="HttpContext"/> for the current request.
+        /// Gets or sets the <see cref="Mvc.ActionContext"/> for the current request.
         /// </summary>
-        public HttpContext HttpContext { get; set; }
+        public ActionContext ActionContext { get; set; }
+
+        /// <summary>
+        /// Gets the <see cref="HttpContext"/> for the current request.
```

Comment:
Yeah, I bet you're right. I'll fix that too. It's something I tend to notice when it autocompletes but isn't the right colour.